R/fledgeling.R

```diff
@@ -0,0 +1,79 @@
+#' fledgeling object
```

Comment:
why not "fledgling" (removing the second e)

Reply:
No strong preference.

Reply:
will rename